### PR TITLE
Add pixels for our main VPN funnels

### PR DESCRIPTION
**Pull request by diegoreymendez**
Task/Issue URL: https://app.asana.com/0/0/1206737255908394/f

macOS PR: https://github.com/duckduckgo/macos-browser/pull/2304
iOS PR: https://github.com/duckduckgo/iOS/pull/2543

## Description

Add pixels to better track all main VPN funnels.  For BSK this means:
- VPN Tunnel Start
- VPN Tunnel Update (change location, etc)

## Testing

See platform-specific PRs for testing instructions.

---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)

### CODE REVIEW

File: Sources/NetworkProtection/PacketTunnelProvider.swift [hunk 1]
```diff
@@ -480,6 +494,12 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
 
     open override func startTunnel(options: [String: NSObject]?, completionHandler: @escaping (Error?) -> Void) {
         Task { @MainActor in
+            os_log("🚀 Options are: %{public}@", String(reflecting: options))
+            os_log("🚀 Error 1: %{public}@", String((TunnelError.startingTunnelWithoutAuthToken as NSError).code))
+            os_log("🚀 Error 2: %{public}@", String((TunnelError.couldNotGenerateTunnelConfiguration(internalError: NSError(domain: "asd", code: 0)) as NSError).code))
```
Comment (samsymons) [inline Sources/NetworkProtection/PacketTunnelProvider.swift:499]: Should we use a proper domain here?

**Review by samsymons — APPROVED** [review]
LGTM besides one comment - thanks!

File: Sources/NetworkProtection/PacketTunnelProvider.swift (hunk 1 above)
Comment (diegoreymendez) [reply Sources/NetworkProtection/PacketTunnelProvider.swift:499]: I'll kill these.

Comment (diegoreymendez) [reply Sources/NetworkProtection/PacketTunnelProvider.swift]: Removed.

**Pull request merged**